cli/smartcontract/smart_contract.go

```diff
@@ -54,7 +54,11 @@ var (
 	}
 	gasFlag = flags.Fixed8Flag{
 		Name:  "gas, g",
-		Usage: "gas to add to the transaction",
+		Usage: "network fee to add to the transaction (prioritizing it)",
+	}
+	sysGasFlag = flags.Fixed8Flag{
```

Comment:
Don't we want to add this flag to transfers?

Reply:
I had this though also. Given `onNEPXX` powers it might be useful.

Reply:
Updated.